### PR TITLE
ENH: Load combined scale and translation transforms to generated PVs

### PR DESCRIPTION
**Pull request by sbelsk**
Closes https://github.com/BrownBiomechanics/SlicerAutoscoperM/issues/133

To reduce clutter for the app users, after the generation of partial volumes in the pre-processing module, only the combined scale and translation transformation layer is loaded and applied onto each PV, instead of the two separate scale and translation trasnforms. I.e., `{}.tfm` is loaded and applied instead of `{}_t.ftm` and `{}_scale.tfm`.

All files are still written to the output directory as part of the PV generation proceess as before.

### CODE REVIEW

**Review by amymmorton — APPROVED** [review]
image


Looks great 
Tested with [BN00124 ](https://drive.google.com/file/d/14dLK307oG974zeKsuA2J2YQWB6nvwJ3n/view?usp=drive_link)

File: AutoscoperM/AutoscoperM.py [hunk 1]
```diff
@@ -768,26 +769,19 @@ def onLoadPV(self):
...
                 return
 
+            # load the combined scale and translation transform for each generated partial volume
             for i in range(len(vols)):
                 nodeName = os.path.splitext(os.path.basename(vols[i]))[0]
                 volumeNode = slicer.util.loadVolume(vols[i])
-                translationTransformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}_t.tfm")
-                scaleTranformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}_scale.tfm")
-                if not os.path.exists(translationTransformFileName):
+                transformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}.tfm")
```
Comment (jcfr) [inline AutoscoperM/AutoscoperM.py:776]: I suggest to keep the check ensuring that `{nodeName}_scale.tfm` and `{nodeName}_t.tfm`

```python
                translationTransformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}_t.tfm")
                if not os.path.exists(translationTransformFileName):
                    raise ValueError(
                        f"Failed to load partial volume {nodeName}: "
                        "Corresponding translation transform file {translationTransformFileName} not found"
                    )

                scaleTranformFileName = os.path.join(mainOutputDir, transformSubDir, f"{nodeName}_scale.tfm")
                if not os.path.exists(scaleTranformFileName):
                    raise ValueError(
                        f"Failed to load partial volume {nodeName}: "
                        "Corresponding scaling transform file {scaleTranformFileName} not found"
                    )
```

File: AutoscoperM/AutoscoperM.py [hunk 2]
```diff
@@ -768,26 +769,19 @@ def onLoadPV(self):
...
+                if not os.path.exists(transformFileName):
                     raise ValueError(
                         f"Failed to load partial volume {nodeName}: "
-                        "Corresponding translation transform file {translationTransformFileName} not found"
+                        "Corresponding translation transform file {transformFileName} not found"
```
Comment (jcfr) [inline AutoscoperM/AutoscoperM.py:780]:
```suggestion
                        "Corresponding combined transform file {transformFileName} not found"
```

**Review by jcfr — CHANGES_REQUESTED** [review]
This will definitely be helpful to reduce the "clutter" and help the user work with the set of loaded transform :100: 

After addressing some minor nitpicks, this will be good for integration :pray:

**Review by jcfr — APPROVED** [review]

**Review by jcfr — APPROVED** [review]

**Pull request merged**